src/js/components/Accordion/accordion.stories.js

```diff
@@ -57,6 +72,28 @@ const SimpleAccordion = props => {
   );
 };
 
+const CustomAccordion = props => {
+  const { animate, multiple, ...rest } = props;
+  return (
+    <Grommet theme={CustomAccordionTheme}>
+      <Box {...rest}>
+        <Accordion animate={animate} multiple>
+          <AccordionPanel label="Panel 1">
+            <Box background="light-2" style={{ height: '200px' }}>
```

Comment:
we should probably avoid using style prop to style things. Box has support for height directly (`height="small"`)

Hm, this was already the case for the other accordions. So handle this as you please ;)